drake/doc/code_review_checklist.rst

```diff
@@ -66,14 +66,16 @@ Are you using pointers well?
 Did you document your API?  Did you comment your complexity?
 ============================================================
 
-- Every nontrivial public symbol must have a doxygen-formatted
+- Every nontrivial public symbol must have a Doxygen-formatted
   comment.
-
 - If you are uncertain of your formatting, consider
-  :ref:`generating the doxygen <documentation-generation-instructions>`
+  :ref:`generating the Doxygen <documentation-generation-instructions>`
   and checking how it looks in a browser.
+- Only use Doxygen comments (``///`` or ``/** */``) on published APIs (public
+  or protected classes and methods).  Code with private access or declared in
+  ``.cc`` files should *not* use the Doxygen format.
```

Comment:
FTR, I would be happy with either this rule as written, xor the opposite rule (always use doxygen for contracts, and non-doxygen for implementation details).  But we were inconsistent and the "don't use doxygen" seemed to carry the day.

I added it to the checklist / styleguide because I tend to use the opposite form (doxygen for contracts), and kept forgetting to undo that before PR, so got defected all the time, and so needed this reminder to do it right.